Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/Concepts.java

```diff
@@ -201,35 +245,40 @@ private JsonValidationResult checkJsonObject(JSONObject jsonObject) {
                     false);
         }
 
-        if (jsonObject.get(JsonFields.CONCEPT_LIST) == null) {
+        if (jsonObject.get(JsonFields.DESCRIPTION) == null) {
             return new JsonValidationResult(
-                    "Error parsing request: please provide a concept list for the concept ('conceptlist' attribute).",
+                    "Error parsing request: please provide a description for the concept ('description' attribute).",
                     jsonObject, false);
         }
-
-        String pos = jsonObject.get(JsonFields.POS).toString();
-        if (!POS.posValues.contains(pos)) {
-            logger.error("Error creating concept from REST call. " + pos + " does not exist.");
-            return new JsonValidationResult("POS '" + pos + "' does not exist.", jsonObject, false);
-        }
-
         return new JsonValidationResult(null, jsonObject, true);
     }
```

Comment:
when you not accept a value then there should be an error, otherwise you're accepting it. Overriding is fine for the 'word' property if it's mentioned in the javadoc, but I would actually returning an error when the user submits a POS different than the one in wordnet, since that might mean the user thought they selected a different wordnet.